ci-operator/config/openshift/release/openshift-release-master__ci-4.18.yaml

```diff
@@ -38,6 +38,7 @@ tests:
     cluster_profile: aws-2
     env:
       ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudCredential CloudControllerManager
+        Ingress
```

Comment:
Also nightlies and periodics (Cluster Bot)?

```console
 $ git grep ADDITIONAL_ENABLED_CAPABILITIES | grep MachineAPI | grep -v Ingress
 ...
 ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml:      ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudCredential CloudControllerManager
ci-operator/config/openshift/release/openshift-release-master__ci-4.14.yaml:      ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI
ci-operator/config/openshift/release/openshift-release-master__ci-4.15.yaml:      ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudCredential
ci-operator/config/openshift/release/openshift-release-master__ci-4.16.yaml:      ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudCredential CloudControllerManager
ci-operator/config/openshift/release/openshift-release-master__ci-4.17.yaml:      ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudCredential CloudControllerManager
ci-operator/config/openshift/release/openshift-release-master__ci-4.18.yaml:      ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudCredential CloudControllerManager
ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml:      ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI
ci-operator/config/openshift/release/openshift-release-master__nightly-4.15.yaml:      ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudCredential
ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml:      ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudCredential CloudControllerManager
ci-operator/config/openshift/release/openshift-release-master__nightly-4.17.yaml:      ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudCredential CloudControllerManager
ci-operator/config/openshift/release/openshift-release-master__nightly-4.18.yaml:      ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudCredential CloudControllerManager
ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml:                ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudCredential CloudControllerManager
ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml:                ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudCredential CloudControllerManager
ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml:                ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudCredential CloudControllerManager
ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml:                ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudCredential CloudControllerManager
```

Looks like QE also took care of this for the `openshift-tests-private` files, but I haven't dug in closely.

Reply:
I'm not entirely sure about clusterbot, thought that was using the CI workflow since it was failing the same way, but now I'm not sure how it's kicking it off since it's not using the `workflow-launch` command

Reply:
>Most of those files are fine except for openshift-release-infra-periodics.yaml but it seems those are older runs of OCP, this should only be for >=4.16

`make update` would update those. 

>now I'm not sure how it's kicking it off since it's not using the workflow-launch command

It would use workflows from https://github.com/openshift/release/blob/master/core-services/ci-chat-bot/workflows-config.yaml

Reply:
Awesome, we should be good then for the workflows, since that's already been updated with  `make update`. 

> >now I'm not sure how it's kicking it off since it's not using the workflow-launch command

> It would use workflows from https://github.com/openshift/release/blob/master/core-services/ci-chat-bot/workflows-config.yaml

Apologies @vrutkovs, I don't see any mention of a `no-capabilities` in that file, I thought that file was only invoked by `workflow-launch <workflow_name_defined_in_workflows-config.yaml>` but the command used to kick off no-capabilities from clusterbot is `launch 4.18 aws,no-capabilities`, if it's using any existing workflow then we should be good here as well.

Reply:
Ah, right, `workflow-launch` would concatenate params and then run `e2e-aws-ovn-no-capabilities` test for that nightly. workflow-configs would be used by `workflow-test` command of the bot

Reply:
I think we're good here then, I can open up a new PR if anything else was missed. Thanks for the input guys!